components/drafts/panel/panel.scss

```diff
@@ -0,0 +1,18 @@
+.Panel {
+    border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
+    background-color: var(--center-channel-bg);
+    border-radius: 4px;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.08);
+    cursor: pointer;
+    transition: box-shadow 0.3s ease-in-out;
+
+    & + .Panel {
+        margin-top: 16px;
+    }
+
+    &:hover {
+        border-color: rgba(var(--center-channel-color-rgb), 0.16);
+        box-shadow: 0 4px 6px rgba(0, 0, 0, 0.12);
```

Comment:
We have `--elevation-2` variables let use that

Reply:
Done